tt_metal/fabric/CMakeLists.txt

```
@@ -8,6 +8,9 @@ set_target_properties(
         VERIFY_INTERFACE_HEADER_SETS
             FALSE
 )
+target_link_options(fabric PRIVATE "LINKER:-Bsymbolic")
+target_compile_options(fabric PUBLIC -Wno-int-to-pointer-cast -fno-semantic-interposition)
```

```suggestion
target_compile_options(
    fabric
    PUBLIC
        -Wno-int-to-pointer-cast
        -fno-semantic-interposition
)
```
Formatting tweak for pre-commit